### PR TITLE
WIP: Add support for Erlang 22

### PR DESCRIPTION
**Pull request by imanhodjaev**
Hi,

In this PR tries to add missing definitions for Erlang 22 to resolve https://github.com/rusterlium/erlang_nif-sys/issues/37. At the moment I've been trying to figure out how to test since I'm quite new to Rust and Nifs as well. Any guidance and help will appreciated.

Thanks.

### CODE REVIEW

File: gen_api.erl [hunk 1]
```diff
@@ -33,6 +33,9 @@ version_opts("2.13") -> [{major,2}, {minor,13}, exception, getenv, time,   % erl
 version_opts("2.14") -> [{major,2}, {minor,14}, exception, getenv, time,   % erlang 21.0
                         dirty_scheduler_opt, nif_2_11, nif_2_12, nif_2_13,
                         nif_2_14];
+version_opts("2.15") -> [{major,2}, {minor,15}, exception, getenv, time,   % erlang 21.0
```
Comment (goertzenator) [inline gen_api.erl]: nitpick:  comment indicates 21 instead of 22

**Comment by goertzenator** [comment]
Thanks for taking this on.  See slightly expanded test writing directions at https://github.com/rusterlium/erlang_nif-sys/wiki/Updating-this-package-for-new-OTP .  Let me know if you need any more guidance.

**Comment by imanhodjaev** [comment]
Thanks for instructions, will now try to actually implement tests.

**Comment by imanhodjaev** [comment]
@goertzenator  I added couple of test cases but not sure if they actually work as expected.

**Comment by goertzenator** [comment]
Travis CI is showing errors on your code.  Are they helping you move forward?

**Comment by imanhodjaev** [comment]
@goertzenator I've been trying to fix them but at the moment unable to clearly understand what is needed. May be I need some help here.

**Comment by lessless** [comment]
Hi guys! Is there any progress on this?

**Comment by imanhodjaev** [comment]
@lessless @goertzenator sorry Im unable to dedicate time at least until september.

**Pull request closed without merging**